src/krb5pa-md5_fmt_plug.c

```diff
@@ -358,9 +357,8 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 
 #ifdef _OPENMP
 #pragma omp parallel for
-	for (i = 0; i < count; i++)
 #endif
-	{
+	for (i = 0; i < count; i++) {
```

Comment:
Duplicate `i = 0` closer to the beginning of the function

Reply:
Fixed now.